posts/2019-8-21-deploy-serverless-cdn-with-serverless-components.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Deploy a Serverless CDN with Serverless Components"
```

Comment:
Suggestion:  Consider a title that explains why this is better than before.

For example, "Easily Deploy A Serverless CDN With Serverless Components"